### DISCUSSION TITLE
希望可以添加更新自选版本；旧版本没有1panel uninstall卸载功能

### DISCUSSION THREAD

**Discussion by liuchenyang0703** [1Panel-dev/1Panel#6660] [Ideas]
### 1Panel 版本

v1.8.5

### 请描述您的需求或者改进建议

希望可以添加更新自选版本，现在是1.8.5版本。升级只能升级到1.10.18版本，升级之后还访问不了页面，服务都正常，也查不到我设置的端口，不知道是不是因为不能跨太大版本升级还是怎么了。
旧版本没有1panel uninstall卸载功能，想直接重新安装1.10.17或18的版本，旧版本没有uninstall，不知道如何卸载。

### 附加信息

![image](https://github.com/user-attachments/assets/9cfb6d17-164e-46c4-9cee-1e906e7ffbc5)

**Comment by wanghe-fit2cloud** [comment]
1. 我们近期已支持 x86_64 架构的离线安装包，若需要安装自定义版本，可以参考使用。
2. 卸载命令为 `1pctl uninstall`。

> 我们在中短期内暂时没有计划支持在线安装自定义版本，望理解。

**Reply by liuchenyang0703** [reply]
好的，谢谢！！！
